spec/support/feature_helpers.rb

```diff
@@ -7,4 +7,8 @@ def assert_current_url(path_with_query, options = {})
       assert_equal Rack::Utils.parse_query(expected.query), Rack::Utils.parse_query(current.query)
     end
   end
+
+  def assert_page_has_content(text)
```

Comment:
Not a blocker, but I feel like we don't need this custom helper method. There are a LOT of places where we're not using this method and doing `expect page.to have_content`

Reply:
Done! (this whole file is removed now)